packages/core/src/components/panel-stack/panelHeader.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+
+import * as React from "react";
+
+import { Icon } from "../icon/icon";
+import { Text } from "../text/text";
+
+import * as Classes from "../../common/classes";
+
+export interface IPanelHeaderProps {
+    /**
+     * The name of the previous panel header in the stack. Will not render unless an onBackClick
+     * handler is defined.
+     */
+    backTitle?: string;
```

Comment:
Oh hum, should this ever be undefined? [title](https://github.com/palantir/blueprint/pull/2642/files#diff-8a6652a4df13b0fb84b0d08566c49beaR37)

Should we allow a panel to be defined without a title?